README.md

```diff
@@ -1,21 +1,16 @@
-# Breach Alerts
+# blurts-server
```

Comment:
Odd change... Bonus points if we can just change this to "Firefox Monitor Server" or something readable, since "blurt" is confusing.